tools/pipelines/templates/include-publish-npm-package-deployment.yml

```
@@ -94,6 +94,14 @@ jobs:
               feedKind: ${{ parameters.feedKind }}
               publishFlags: ${{ parameters.publishFlags }}
 
-          - template: /tools/pipelines/templates/include-git-tag-steps.yml@self
-            parameters:
-              tagName: ${{ parameters.tagName }}
+- job: TagRelease
+  displayName: Tag Release
+  dependsOn: publish_${{ replace(parameters.environment, '-', '_') }}
```

And similar note in the one above? We could also make it a parameter but feels a bit funky since `environment` already is one.
```suggestion
  # Note: must be kept in sync with the name of the deployment job above
  dependsOn: publish_${{ replace(parameters.environment, '-', '_') }}
```